tests/Expected/Business/testJsonSerialize.json

```diff
@@ -0,0 +1,42 @@
+{
+  "days": {
+    "1": {
+      "dayOfWeek": 1,
+      "openingIntervals": [
+        {
+          "start": {
+            "hours": 9,
+            "minutes": 0,
+            "seconds": 0
+          },
+          "end": {
+            "hours": 13,
+            "minutes": 0,
+            "seconds": 0
+          }
+        },
+        {
+          "start": {
+            "hours": 14,
+            "minutes": 0,
+            "seconds": 0
+          },
+          "end": {
+            "hours": 17,
+            "minutes": 0,
+            "seconds": 0
+          }
+        }
+      ]
+    },
+    "5": {
+      "dayOfWeek": 5,
+      "openingIntervals": []
```

Comment:
I think it should not be empty as per the `SpecialDay` definition https://github.com/florianv/business/pull/20/files#diff-53a49fd8d2d6d55e33a12b52fc2601b8R508